### ISSUE TITLE
PEAR.Functions.FunctionCallSignature sniff not checking all function calls

### ISSUE THREAD

**Issue by aik099** [squizlabs/PHP_CodeSniffer#589]
Code fragment:
```php
if ( isset( $validation_data['address']) ) {
    $a = 5;
}

isset( $validation_data['address']);
```

If appears, that PEAR.Functions.FunctionCallSignature sniff isn't checking that there is 0 spaces after opening parenthesis of "isset" function call, when it's placed within a control structure.

After some digging I've found, that this has something to do with "isset" function call specifically. If I replace "isset" with "array_values" or "some_func", then all is reported/fixed properly.

**Comment by aik099** [comment]
Ha, apparently to PHP tokenizer "isset" isn't a function, because it's tokenized as `T_ISSET` token and therefore isn't considered as function call by any of the sniffs.

**Comment by JDGrimes** [comment]
- T_EMPTY - empty( ... );

**Comment by aik099** [comment]
Thanks @JDGrimes , I've updated my original comment.

**Comment by gsherwood** [comment]
Yep, those special tokens are a real pain. This is fixed now.

**Comment by pfrenssen** [comment]
`isset()` is not a function, but a language construct.

```
$ php -a
php > $function = 'isset';
php > $function('test');
PHP Warning:  Uncaught Error: Call to undefined function isset() in php shell code:1
```

**Comment by gsherwood** [comment]
> isset() is not a function, but a language construct.

In this case, it is called like a function and so the same formatting rules apply.
